cobigen-cli/cli-systemtest/src/test/java/com/devonfw/cobigen/cli/systemtest/GenerateCommandTest.java

```diff
@@ -74,22 +73,21 @@ public void generateFromEntityTest() throws Exception {
    * @throws Exception test fails
    */
   @Test
-  @Ignore
   public void generateFromTemplatesJarWithUtilClassDependencyTest() throws Exception {
 
     FileUtils.copyDirectory(new File(testFileRootPath + "templatesproject"), this.tmpProject.toFile());
     File baseProject = this.tmpProject.resolve("maven.project/core/").toFile();
     File templatesProject = this.tmpProject.resolve("templates-devon4j/target/templates-devon4j-dev-SNAPSHOT.jar")
         .toFile();
 
-    String args[] = new String[6];
+    String args[] = new String[7];
     args[0] = "generate";
     args[1] = this.entityInputFile.getAbsolutePath();
     args[2] = "--increments";
     args[3] = "tos";
     args[4] = "-tp";
     args[5] = templatesProject.getAbsolutePath();
-
+    args[6] = "--force-old-templates";
     execute(args, false);
```

Comment:
better add this param to AbstractCLITest and overwrite the execute method with another force param instead (similar like useDevDemplates).